code/apps/Managed Software Center/Managed Software Center/Controllers/MainWindowController.swift

```diff
@@ -1070,7 +1088,8 @@ class MainWindowController: NSWindowController, NSWindowDelegate, WKNavigationDe
                 decisionHandler(.cancel)
                 return
             }
-            if scheme == "mailto" || scheme == "http" || scheme == "https" {
+            
+            if scheme == "mailto" {
```

Comment:
Does this then change the behavior for http/https links from other contexts, like in item descriptions?

Reply:
I'm not quite sure yet.
I'm still testing when and how this part triggers.

Reply:
Custom links that don't have `target="_blank"` would go though this code path, I'd think (Links in client customization, like the footer, the banners, and the (former) sidebar-now-link-strip)